docs/models.rst

```diff
@@ -373,4 +414,4 @@ parameters:
     >>> source = sncosmo.SALT2Source(modeldir='/path/to/dir',
     ...                              m0file='mytemplate0file.dat')
 
-See `~sncosmo.SALT2Source` for more details.
+See  for more details.
```

Comment:
Why did this change?

Reply:
Sorry, probably done a ctr+X instead of ctrl+C. Fixed.